repos/spack_repo/builtin/packages/latte/package.py

```diff
@@ -15,28 +15,34 @@ class Latte(CMakePackage):
     url = "https://github.com/lanl/latte/tarball/v1.2.1"
     git = "https://github.com/lanl/latte.git"
 
+    maintainers("jeanlucf22", "finkeljos")
+
     tags = ["ecp", "ecp-apps"]
 
     license("LGPL-2.0-or-later")
 
     version("master", branch="master")
+    version("lattepy", branch="lattepy")
     version("1.2.2", sha256="ab1346939dbd70ffd89c5e5bf8d24946cb3655dc25b203bec7fc59c6c63e4c79")
     version("1.2.1", sha256="a21dda5ebdcefa56e9ff7296d74ef03f89c200d2e110a02af7a84612668bf702")
     version("1.0.1", sha256="67b2957639ad8e36b69bc6ea9a13085183a881562af9ca6d2b90b412ff073789")
 
-    variant("mpi", default=True, description="Build with mpi")
-    variant("progress", default=False, description="Use progress for fast")
-    variant("shared", default=True, description="Build shared libs")
-
     depends_on("c", type="build")  # generated
     depends_on("fortran", type="build")  # generated
-
     depends_on("cmake@3.1:", type="build")
     depends_on("blas")
     depends_on("lapack")
+
+    variant("interface", default=False, description="Build with interfacing to use with sedacs")
+
+    variant("mpi", default=True, description="Build with mpi")
     depends_on("mpi", when="+mpi")
+
+    variant("progress", default=False, description="Use progress for fast solvers")
     depends_on("qmd-progress", when="+progress")
 
+    variant("shared", default=True, description="Build shared libs")
```

Comment:
See comment on `sedacs` regarding ordering of `variant` and `depends_on directives`.